packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -32,9 +32,6 @@ import {
 } from '../sources/migration'
```

Comment:
Just to clarify for reviewers: every file change below this line is reverted from https://github.com/cypress-io/cypress/pull/22118 and you don't need to review it to review this PR.